SConscript.mstruct.version

```diff
@@ -8,8 +8,8 @@ def config_h_build(target, source, env):
     config_h_defines = env.Dictionary()
 
     for a_target, a_source in zip(target, source):
-        config_h = file(str(a_target), "w")
-        config_h_in = file(str(a_source), "r")
+        config_h = open(str(a_target), "w")
+        config_h_in = open(str(a_source), "r")
         config_h.write(config_h_in.read() % config_h_defines)
         config_h_in.close()
         config_h.close()
```

Comment:
Consider using context managers here instead of manually opening and closing
```suggestion
        with open(str(a_target), "w") as config_h:
            with open(str(a_source), "r") as config_h_in:
                config_h.write(config_h_in.read() % config_h_defines)
```